core/src/test/java/org/elasticsearch/index/translog/TranslogTests.java

```diff
@@ -1993,4 +2003,46 @@ public void testPendingDelete() throws IOException {
     public static Translog.Location randomTranslogLocation() {
         return new Translog.Location(randomLong(), randomLong(), randomInt());
     }
+
+    public void testTranslogOpSerialization() throws Exception {
+        BytesReference B_1 = new BytesArray(new byte[]{1});
+        SeqNoFieldMapper.SequenceID seqID = SeqNoFieldMapper.SequenceID.emptySeqID();
+        long randomSeqNum = randomBoolean() ? SequenceNumbersService.UNASSIGNED_SEQ_NO : randomNonNegativeLong();
```

Comment:
yes. I should have been more precise - the usage of SequenceNumbersService.UNASSIGNED_SEQ_NO is only relevant due to a BWC aspect.